### PR TITLE
Bug 1627312 - Fix undocumented pieces in Swift

### PR DESCRIPTION
**Pull request by badboy**
```
100% documentation coverage with 0 undocumented symbols
```

### CODE REVIEW

File: glean-core/ios/Glean/Metrics/DistributionData.swift [hunk 1]
```diff
@@ -14,6 +14,9 @@ public class DistributionData {
     let values: [UInt64: UInt64]
     let sum: UInt64
 
+    /// Parse the distribution data from the given JSON string.
+    ///
+    /// If the string is not valid JSON or data is missing `nil` is returned.
```
Comment (travis79) [inline glean-core/ios/Glean/Metrics/DistributionData.swift:19]: I'm not sure it's entirely necessary for all functions but you omitted the `- parameters` section for this function.

File: glean-core/ios/Glean/Metrics/Ping.swift [hunk 2]
```diff
@@ -25,6 +29,7 @@ public class PingBase {
     var handle: UInt64
     var name: String
 
+    /// Create a new Ping with the given name.
```
Comment (travis79) [inline glean-core/ios/Glean/Metrics/Ping.swift:32]: Again, not sure if we really need it here but this is also missing `- parameters` section.

File: glean-core/ios/Glean/Metrics/DistributionData.swift (hunk 1 above)
Comment (badboy) [reply glean-core/ios/Glean/Metrics/DistributionData.swift:19]: yeah I was like "do I really need to document a very well named parameter?"

Comment (travis79) [reply glean-core/ios/Glean/Metrics/DistributionData.swift:19]: Good enough explanation for me :))

**Pull request merged**